### ISSUE TITLE
Batch download icons and set automatically

### ISSUE THREAD

**Issue by teemue** [MacPass/MacPass#975]
Currently I have to go through every entry to download icon for each item. Plus, I need to select downloaded icons manually. This could be automated.

**Comment by mstarke** [comment]
Indeed. But first the Icon-downloader needs to get much more sophisticated.
